app/src/main/java/se/inera/intyg/minaintyg/util/html/HTMLUtility.java

```diff
@@ -0,0 +1,23 @@
+package se.inera.intyg.minaintyg.util.html;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public final class HTMLUtility {
```

Comment:
Add private constructors and throw new IllegalStateException("Utility class"); if used.